src/app/@ansyn/ansyn/modules/menu-items/results/components/results-table/results-table.component.ts

```diff
@@ -95,8 +98,15 @@ export class ResultsTableComponent implements OnInit, OnDestroy {
 	ngOnInit(): void {
 	}
 
+	paginateOverlays(overlays: IOverlayDrop[]) {
+		const pagination = 15;
```

Comment:
maybe is better to save it on configuration,
make our customer the ability to change the pagination